test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -718,6 +716,15 @@ export class TestRunner {
     if (other)
       testName += `_${other}`;
 
+    // NB: We removed the 3 lighting properties from ViewFlags, replacing them with a single `lighting` property.
```

Comment:
@DStradley the view flags string will be the last thing in the test name except for the optional test number used to differentiate duplicate test names. That last bit always begins with "---". So your should be able to append "-lit" to any test names that contain "_SmoothShade" and *none* of "+slL", "+cmL", or "+scL".